full/Dockerfile

```diff
@@ -1,3 +1,4 @@
+# Dazzle do not supports ARG TRIGGER_REBUILD=2
```

Comment:
Dazzle supports `ENV`, but decides to fail early when it sees `ARG`.

I've already fixed that in https://github.com/gitpod-io/dazzle/pull/6

Reply:
This is the way I found to avoid using an old workspace-base cache during workspace-full build